cli/bench/gzip/compression.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+let [total, count] = typeof Deno !== "undefined"
+  ? Deno.args
+  : [process.argv[2], process.argv[3]];
+
+total = total ? parseInt(total, 0) : 3;
+count = count ? parseInt(count, 10) : 100;
+
+async function bench(name, fun) {
+  if (total === 3) console.log(name);
+  const start = Date.now();
+  for (let i = 0; i < count; i++) await fun();
+  const elapsed = Date.now() - start;
+  const rate = Math.floor(count / (elapsed / 1000));
+  console.log(`time ${elapsed} ms rate ${rate}`);
+
+  if (--total) await bench(null, fun);
+  else total = 3;
+}
+
+function compress(buffer) {
+  const cs = new CompressionStream("gzip");
+  const writer = cs.writable.getWriter();
+  writer.write(buffer);
+  writer.close();
+  return (new Response(cs.readable).arrayBuffer());
+}
+
+function decompress(buffer) {
+  const ds = new DecompressionStream("gzip");
+  const writer = ds.writable.getWriter();
+  writer.write(buffer);
+  writer.close();
+  return (new Response(ds.readable).arrayBuffer());
+}
+
+const data = typeof Deno !== "undefined"
+  ? Deno.readFileSync("cli/bench/gzip/uncompressed.png")
```

Comment:
Can you replace the current image with one of the images from the Deno blog post?